fix: setup-imir action fails to parse latest release version

**Problem:**
Badge sync workflow fails on `Setup IMIR` step with exit code 1.

**Root cause:**
In `.github/actions/setup-imir/action.yml` line 33, the script uses incorrect parsing:

```bash
VERSION=$(gh release list --repo ${{ github.repository }} --limit 100 | grep '^v' | head -1 | awk '{print $1}')
```

**Issue:**
`gh release list` output format is tab-separated:
```
TITLE\tLATEST_LABEL\tTAG\tDATE
IMIR v0.1.5\tLatest\tv0.1.5\t2025-10-10T07:33:56Z
```

Current script:
- Uses `grep '^v'` but lines start with "IMIR", not "v"
- Takes first column (title) instead of third column (tag)

**Solution:**
Replace with simpler, more reliable approach:

```bash
VERSION=$(gh release view --repo ${{ github.repository }} --json tagName -q .tagName)
```

Or use correct awk column:
```bash
VERSION=$(gh release list --repo ${{ github.repository }} --limit 1 | awk '{print $3}')
```

**Evidence:**
- Workflow run: https://github.com/RAprogramm/infra-metrics-insight-renderer/actions/runs/18399692095
- Error: Process completed with exit code 1
- Failed step: Setup IMIR → Determine version to download

**Impact:**
Badge sync workflow cannot run, blocking automatic badge updates.

--- a/.github/actions/setup-imir/action.yml
+++ b/.github/actions/setup-imir/action.yml
@@ -30,7 +30,7 @@ runs:
         if [ -n "${{ inputs.version }}" ]; then
           echo "version=${{ inputs.version }}" >> $GITHUB_OUTPUT
         else
-          VERSION=$(gh release list --repo ${{ github.repository }} --limit 100 | grep '^v' | head -1 | awk '{print $1}')
+          VERSION=$(gh release view --repo ${{ github.repository }} --json tagName -q .tagName)
           echo "version=${VERSION}" >> $GITHUB_OUTPUT
         fi
       env:

